build.sbt

```diff
@@ -1 +1,3 @@
-scalaVersion := "2.11.7"
\ No newline at end of file
+scalaVersion := "2.11.7"
+
+libraryDependencies := Seq ("org.scalaz"  % "scalaz-core_2.9.2"   % "6.0.4")
```

Comment:
Yea. That's true. Thanks for bringing it up. It compiled but we may not be able to import it once we start using it. Il change 

Sent from my iPhone

> On 21-Nov-2015, at 3:33 pm, Shyam Sundar <notifications@github.com> wrote:
> 
> In build.sbt:
> 
> > \ No newline at end of file
> > +scalaVersion := "2.11.7"
> > +
> > +libraryDependencies := Seq ("org.scalaz"  % "scalaz-core_2.9.2"   % "6.0.4")
> That looks like an old version of scalaz and why scala 2.9, wouldn't it be binary incompatible?
> Maybe libraryDependencies += "org.scalaz" %% "scalaz-core" % "7.1.5" is what you're after.
> 
> —
> Reply to this email directly or view it on GitHub.
>